### PR TITLE
FAT-10294-C422050

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://folio-org.atlassian.net/browse/FAT-10294
***
https://foliotest.testrail.io/index.php?/cases/view/422050
***
Screenshot 2024-09-24 220933
***

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -146,7 +146,7 @@ module.exports = defineConfig({
 
       return result;
     },
-    baseUrl: 'https://folio-testing-cypress-diku.ci.folio.org',
+    baseUrl: 'https://folio-testing-sprint-cs00000int.ci.folio.org',
```
Comment (eremv) [inline cypress.config.js:149]: Are you sure that change is required?

File: cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+import uuid from 'uuid';
+import Affiliations, { tenantNames } from '../../../../support/dictionary/affiliations';
+import Permissions from '../../../../support/dictionary/permissions';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import ConsortiumManager from '../../../../support/fragments/settings/consortium-manager/consortium-manager';
+import TopMenu from '../../../../support/fragments/topMenu';
+import DateTools from '../../../../support/utils/dateTools';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+// import InventorySearchAndFilter from '../../../../support/fragments/inventory/inventorySearchAndFilter';
```
Comment (eremv) [inline cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js:11]: Please remove comments if they're not nessesary

File: cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        cy.login(user.username, user.password, {
+          path: TopMenu.inventoryPath,
+          waiter: InventoryInstances.waitContentLoading,
+        });
+        ConsortiumManager.switchActiveAffiliation(tenantNames.central, tenantNames.college);
+        ConsortiumManager.checkCurrentTenantInTopMenu(tenantNames.college);
+      });
+    });
+
+    // after('Delete test data', () => {
```
Comment (eremv) [inline cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js:59]: And here

**Review by eremv — CHANGES_REQUESTED** [review]

File: cypress.config.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress.config.js]: I've fixed it

File: cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js]: I've deleted

**Review by eremv — APPROVED** [review]

**Review by eremv — APPROVED** [review]

**Review by Sherzod-Kenjaev — APPROVED** [review]

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js (hunk 3 above)
Comment (TetianaParanich) [reply cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js]: Done

**Pull request merged**